vendors/index.js

```diff
@@ -1 +1 @@
-import 'flag-icon-css/css/flag-icon.css'
\ No newline at end of file
+import "flag-icons/css/flag-icon.min.css";
```

Comment:
this breaks stuff as it is `flag-icons.min.css` amended and published a new version (#24 ), feel free to do another PR with the updated changes, thanks for #21